tests/netperf_test.py

```diff
@@ -13,35 +13,40 @@
 import time
 import os
 from devices import board, wan, lan, wlan, prompt
-
 def install_netperf(device):
     # Check version
     device.sendline('\nnetperf -V')
     try:
-        device.expect('Netperf version 2.4', timeout=10)
+        device.expect('Netperf version 2.6', timeout=10)
```

Comment:
We should probably do 4 or 6... or better yet just 'Netperf version 2.' unless we think it will only work with one version or the other.